_posts/2020-04-21-a-token-of-our-affection-uswds-2.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "A token of our affection - A field guide to USWDS 2"
+date: 2020-04-21
+authors:
+- christine-bath
+- laura-gerhardt
+tags:
+- web design system
+- lessons learned
+- cloud.gov
+excerpt: "We recently worked with the cloud.gov team to update their public site, cloud.gov, to United States Web Design System 2. The USWDS provided concepts we were able to use to translate designs into code a lot faster and deliver higher fidelity results once we understood how to use them."
+---
+
+We recently worked with the cloud.gov team to update their public site, [cloud.gov](https://cloud.gov/), to [United States Web Design System 2 (USWDS 2)](https://designsystem.digital.gov/). This was our first time digging into Version 2 of the system.
+Using the latest version of the web design system lets us easily correct some usability concerns with the site, and work towards compliance with the [21st Century Idea Act web design standards](https://digital.gov/resources/21st-century-integrated-digital-experience-act/).
+
+Designers and developers use different tools and vocabulary to deliver
+government digital services and the USWDS provided concepts we were able
+to use to translate designs into code a lot faster and deliver higher
+fidelity results once we understood how to use them.
+
+## Theming
+
+USWDS provides variables, scss, and structure for custom themes that lets teams customize their government websites. Themes allowed us to use the consistent development and design patterns of the USWDS to showcase cloud.gov’s unique brand identity. We learned that these themes are extensive and allow lots of flexibility to design sites that reflect an agency’s visual style. We leveraged the default user experience behavior of the USWDS 2, while updating the colors and typography to match the brand identity of cloud.gov.
```

Comment:
Would it make more sense to say "Themes allowed us to use the consistent development and design patterns of the USWDS _while_ showcasing cloud.gov’s unique brand identity."?